src/transaction.js

```diff
@@ -720,10 +726,13 @@ class Transaction extends TransactionRequest {
    * @return {object}
    */
   static createDeadlineError_(err) {
-    return extend({}, err, {
-      code: 4,
-      message: 'Deadline for Transaction exceeded.',
-    });
+    let apiError = new common.util.ApiError(
```

Comment:
Looking at [the code](https://github.com/googleapis/nodejs-common/blob/4a3451f9a51b55b0a655f5f4990dc2564616cbe2/src/util.ts#L172), I think the intended way to call this is:

```js
let apiError = new common.util.ApiError({
  message: 'Deadline for Transaction exceeded.',
  code: DEADLINE_EXCEEDED,
  errors: [err],
});
```

This would return (from REPL):

```js
{ Error: Deadline for Transaction exceeded. - Original error message.
  code: 4,
  errors:
   [ Error: Original error message. ],
  response: undefined,
  message: 'Deadline for Transaction exceeded. - Blah.' }
```